model-optimizer/extensions/front/DropoutWithRandomUniformReplacer.py

```diff
@@ -1,73 +0,0 @@
-# Copyright (C) 2018-2021 Intel Corporation
-# SPDX-License-Identifier: Apache-2.0
-
-import logging as log
-import numpy as np
-
-from mo.front.common.replacement import FrontReplacementSubgraph
-from mo.front.tf.graph_utils import create_op_with_const_inputs
-from mo.graph.graph import Graph, Node, rename_nodes
-from mo.middle.pattern_match import check_value
-from mo.ops.broadcast import Broadcast
-
-
-class DropoutWithRandomUniformReplacer(FrontReplacementSubgraph):
```

Comment:
I have just realized that removal of this transformation may have broken some of the existing models. Previously the sub-graph with RandomUniform was replaced with Broadcast operation which is supported with all plugins. Now, the model is generated with a RandomUniform which is not supported by all plugins. So we potentially broke inference with some plugins.

1. Let's find all models that we have with this pattern to understand the impact. 
2. Probably, we should keep with transformation enabled by default but do not perform any changes and just show the user message (if it detected the sub-graph) on how to force transformation to perform actual graph modification. So by default it will convert the model as-is (with RandomUniform operation), but user will still be able to generate the model as before. We need to find a convenient way to do this.

@rkazants , @jane-intel, what do you think?

Reply:
Probably we need to write an nGraph transformation which will fuse DropoutWithRandomUniform sub-graph into a Broadcast op as it was previously done in the MO. So the plugin which does not support RandomUniform can trigger it. This looks like the best option. Anyway, still wait for comments from @rkazants and @jane-intel

Reply:
Here are models from Benchmark that have this pattern:
3d-pose-baseline
facenet-20180408-102900

I implemented nGraph transformation that replaces RandomUniform with Broadcast in possible dropout subgraph and checked on these models. Transformation also fixed failing test in e2e. 
Please review the transformation.